pygmt/datasets/earth_age.py

```diff
@@ -91,7 +91,7 @@ def load_earth_age(resolution="01d", region=None, registration=None):
     ...     region=[120, 160, 30, 60],
     ...     registration="gridline",
     ... )
-    """
+    """  # noqa: W505
```

Comment:
In commit ece073013fc85887a99ac7ee4b720639dd0867cb, I had to move these [`noqa: W505` (doc-too-long)](https://docs.astral.sh/ruff/rules/doc-line-too-long) comments (mostly added in #2728) to behind the triple quotes instead of inline, because `ruff` requires it to be like so. See https://github.com/astral-sh/ruff/issues/3972 and https://docs.astral.sh/ruff/configuration/#error-suppression.